extra/2-piping.js

```diff
@@ -16,26 +16,32 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  return a+b;
 }
 
-function multiply() {
-
+function multiply(a, b) {
+  return a*b;
 }
 
-function format() {
-
+function format(a) {
+  return "£"+a; 
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+// too many functions, difficult to understand;
+format (multiply(add(startingValue, 10), 2));
```

Comment:
Looks like the test is failing because we're not assigning a value to the variable badCode. Can you think of how to fix this?